lib/presentation/widgets/expenses_icons.dart

```diff
@@ -0,0 +1,16 @@
+import 'package:trip_tally/presentation/widgets/custom_icon_button.dart';
+
+import '../theme/app_paths.dart';
+
+final expensesIcons = [
+  CustomIconButton(icon: AppPaths.biggerAirplane, onPressed: () {}),
+  CustomIconButton(icon: AppPaths.cycle, onPressed: () {}),
+  CustomIconButton(icon: AppPaths.cutlery, onPressed: () {}),
+  CustomIconButton(icon: AppPaths.food, onPressed: () {}),
+  CustomIconButton(icon: AppPaths.hotel, onPressed: () {}),
+  CustomIconButton(icon: AppPaths.taxi, onPressed: () {}),
+  CustomIconButton(icon: AppPaths.ticket, onPressed: () {}),
+  CustomIconButton(icon: AppPaths.car, onPressed: () {}),
+  CustomIconButton(icon: AppPaths.bus, onPressed: () {}),
+  CustomIconButton(icon: AppPaths.ship, onPressed: () {}),
+];
```

Comment:
If you keep widgets like this it will impact performance. Everytime parent rebuilds all these widgets rebuilds. It's better to keep the just the icons in the variable not all the widgets